RadixWallet/Features/SettingsFeature/Preferences/Preferences+View.swift

```diff
@@ -0,0 +1,182 @@
+private typealias S = L10n.Preferences
+
+extension Preferences.State {
+	var viewState: Preferences.ViewState {
+		let isDeveloperModeEnabled = appPreferences?.security.isDeveloperModeEnabled ?? false
+		#if DEBUG
+		return .init(
+			isDeveloperModeEnabled: isDeveloperModeEnabled,
+			exportLogsUrl: exportLogsUrl
+		)
+		#else
+		return .init(
+			isDeveloperModeEnabled: isDeveloperModeEnabled
+		)
+		#endif
+	}
+}
+
+// MARK: - Preferences.View
+
+public extension Preferences {
+	struct ViewState: Equatable {
+		let isDeveloperModeEnabled: Bool
+		#if DEBUG
+		let exportLogsUrl: URL?
+		init(isDeveloperModeEnabled: Bool, exportLogsUrl: URL?) {
+			self.isDeveloperModeEnabled = isDeveloperModeEnabled
+			self.exportLogsUrl = exportLogsUrl
+		}
+		#else
+		init(isDeveloperModeEnabled: Bool) {
+			self.isDeveloperModeEnabled = isDeveloperModeEnabled
+		}
+		#endif
+	}
+
+	@MainActor
+	struct View: SwiftUI.View {
+		private let store: StoreOf<Preferences>
+
+		public init(store: StoreOf<Preferences>) {
+			self.store = store
+		}
+
+		public var body: some SwiftUI.View {
+			content
+				.setUpNavigationBar(title: S.title)
+				.tint(.app.gray1)
+				.foregroundColor(.app.gray1)
+				.presentsLoadingViewOverlay()
+				.destinations(with: store)
+		}
+	}
+}
+
+extension Preferences.View {
+	@MainActor
+	private var content: some View {
+		WithViewStore(store, observe: \.viewState, send: { .view($0) }) { viewStore in
+			ScrollView {
+				VStack(spacing: .zero) {
+					ForEach(rows(viewStore: viewStore)) { kind in
+						kind.build(viewStore: viewStore)
+					}
+				}
+			}
+			.background(Color.app.gray4)
+			.onAppear {
+				viewStore.send(.appeared)
+			}
+			#if DEBUG
+			.sheet(item: viewStore.binding(get: \.exportLogsUrl, send: { _ in .exportLogsDismissed })) { item in
+					ShareView(items: [item])
+				}
+			#endif
+		}
+	}
+
+	@MainActor
+	private func rows(viewStore: ViewStoreOf<Preferences>) -> [SettingsRow<Preferences>] {
+		var visibleRows = normalRows(viewStore: viewStore)
+		#if DEBUG
+		visibleRows.append(.model(
+			title: "Export logs",
+			subtitle: "Export and save debugging logs",
+			icon: .asset(AssetResource.appSettings),
+			action: .exportLogsButtonTapped
+		))
+		#endif
+		return visibleRows
+	}
+
+	@MainActor
+	private func normalRows(viewStore: ViewStoreOf<Preferences>) -> [SettingsRow<Preferences>] {
+		[
+			.separator,
+			.model(
+				title: S.DepositGuarantees.title,
+				subtitle: S.DepositGuarantees.subtitle,
+				icon: .asset(AssetResource.depositGuarantees),
+				action: .depositGuaranteesButtonTapped
+			),
+			.model(
+				title: S.HiddenEntities.title,
+				subtitle: S.HiddenEntities.subtitle,
+				icon: .systemImage("eye.fill"),
+				action: .hiddenEntitiesButtonTapped
+			),
+			.header(S.advancedPreferences),
+			.model(
+				title: S.gateways,
+				icon: .asset(AssetResource.gateway),
+				action: .gatewaysButtonTapped
+			),
+			.custom(developerMode(viewStore: viewStore), id: "developerMode"),
```

Comment:
What is custom and why does it take an AnyView? Let's make the id at least a private let.

Reply:
If you go into `SettingsRow.custom` it explains what custom is. It allows to adding a different type of row that one defined by `PlainListRow`.